src/components/School.js

```diff
@@ -41,7 +40,7 @@ const School = () => {
 
             <h1>My School name is : {schoolName} </h1>
             {/* <UserTitle title={title} /> */}
-            <StudentList />
+            <StudentList removeStudent={removeStudent}/>
```

Comment:
you don't need to pass removeStudent as props from here .we can directly use  removeStudent  from context in student component.